src/utility.js

```diff
@@ -0,0 +1,14 @@
+export const pipe = (...functions) =>
+  (x) =>
+    functions.reduce(
+      (acc, fn) => fn(acc),
+      x,
+    );
```

Comment:
I think I expect more brackets on this? That might be my linting settings though

Reply:
I found this function on [MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/reduce#function_composition_enabling_piping). Adding brackets causes it to break. Exactly why I am not exactly sure.